src/ensembl/src/content/app/browser/drawer/drawer-views/gene-summary/GeneSummary.tsx

```diff
@@ -122,27 +128,50 @@ const GeneSummary = () => {
         </div>
       </div>
 
-      <div className={`${styles.row} ${styles.spaceAbove}`}>
+      <div className={rowClasses}>
         <div className={styles.label}>Gene name</div>
         <div className={styles.value}>{gene.name}</div>
       </div>
 
       {gene.alternative_symbols.length > 0 && (
-        <div className={`${styles.row} ${styles.spaceAbove}`}>
+        <div className={rowClasses}>
           <div className={styles.label}>Synonyms</div>
           <div className={styles.value}>
             {gene.alternative_symbols.join(', ')}
           </div>
         </div>
       )}
 
-      <div className={`${styles.row} ${styles.spaceAbove}`}>
+      <div className={rowClasses}>
         <div className={styles.value}>
           {`${gene.transcripts.length} transcripts`}
         </div>
       </div>
 
-      <div className={`${styles.row} ${styles.spaceAbove}`}>
+      <div className={classNames(rowClasses, styles.downloadRow)}>
+        <div className={styles.value}>
+          <div
+            className={styles.downloadLink}
+            onClick={() => showDownload(!shouldShowDownload)}
+          >
+            Download
+          </div>
+          {shouldShowDownload && (
+            <div className={styles.downloadWrapper}>
+              <InstantDownloadGene
+                genomeId={ensObjectGene.genome_id}
+                gene={{ id: gene.unversioned_stable_id }}
```

Comment:
In https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-785 (I don't think he's pushed his changes to github yet) @jyothishnt is adding versions to stable ids both in the UI and in the headings of the downloaded sequence. Which means that he is passing a regular `stable_id` to the instant download component, and then does various tricks to fetch the unversioned stable id from thoas that he will need for fetching genomic sequences from REST.

I wonder if the approach in both PRs should be similar.

Reply:
Yes agree they should be similar. I'll check him PR and try to replicate that here.

Reply:
I am now passing the versioned stabled ID to `InstantDownloadGene`. The unversioned stabled IDs are retrieved inside the component from Thoas to request genomic sequences via REST.

✅